great_expectations/expectations/core/expect_table_columns_to_match_set.py

```diff
@@ -170,7 +170,9 @@ class ExpectTableColumnsToMatchSet(BatchExpectation):
     column_set: Union[list, set, SuiteParameterDict, None] = pydantic.Field(
         description=COLUMN_SET_DESCRIPTION
     )
-    exact_match: Union[bool, None] = pydantic.Field(description=EXACT_MATCH_DESCRIPTION)
+    exact_match: Union[bool, None] = pydantic.Field(
+        default=None, description=EXACT_MATCH_DESCRIPTION
```

Comment:
This is to make mypy happy, and keeps behavior the same as before. It's actually [a bug to not default to True](https://greatexpectations.io/expectations/expect_table_columns_to_match_set), though. I want to fix that in a separate PR so we can separate the [MAINTENANCE] vs [BUGFIX] labels.